hibernate-core/src/test/java/org/hibernate/orm/test/cdi/general/nonregistrymanaged/NonRegistryManagedBeanConsumingIntegrator.java

```diff
@@ -69,68 +69,80 @@ public void integrate(Metadata metadata, SessionFactoryImplementor sessionFactor
 		applicationScopedBean1 = (ContainedBeanImplementor) beanContainer.getBean(
 				TheApplicationScopedBean.class,
 				this,
-				fallbackBeanInstanceProducer
+				fallbackBeanInstanceProducer,
+				true
```

Comment:
I can say for sure that existing integrators using this API (i.e. Hibernate Search) do not, at the moment, use `true`.

Since these tests are specifically about checking compatibility with Hibernate Search, it's probably better to keep calling the old method without the `cdiRequiredIfAvailable` flag?